content/about.html

```diff
@@ -79,7 +78,7 @@
                 <li><a tabindex="-1" href="contribute.html">Get Involved</a></li>
                 <li><a tabindex="-1" href="developers.html">Developers</a></li>
                 <li><a tabindex="-1" href="mailing-lists.html">Mailing Lists</a></li>
-                <li><a tabindex="-1" href="http://lanyrd.com/topics/apache-cloudstack/" target="_blank">Events &amp; Meetups <span class="glyphicon glyphicon-share-alt pull-right"></span></a></li>
```

Comment:
I was going to say exactly that. At least for me, it feels something that we stopped using and maintaining a long time ago. Could not we use the system/website of CCC for spread the word about user group meeting too? Then, we centralize information.